sonar-fsharp-plugin/src/main/resources/com/sonar/sqale/fsharp-fslint-model.xml

```diff
@@ -117,19 +117,6 @@
                     <txt>mn</txt>
                 </prop>
             </chc>
-            <chc>

```

Comment:
@jmecosta Do you remember for which reason this file is required? Didn't find where it is used.

Reply:
that's the old sqale model file, it provides basic values for calculation of technical debt. most likely those or part of those are decrecated with the introduction of the smell/bug/vulnerability but I have checked how other plugins are doing that. For now it seems it will still take the estimates and if it works I suggest leave it like that for now. needs proper checking later

Reply:
I was curious as the removed rules where in this file - and there are lots of rules which we will need to introduce tofully support the current FSharpLint.